.babelrc

```diff
@@ -1,34 +1,6 @@
 {
-  "env": {
-    "development": {
-      "presets": ["es2015"],
-      "plugins": [
-        "transform-object-rest-spread",
-        "transform-async-to-generator",
-        "transform-runtime"
-      ],
-    },
-    "default": {
-      "presets": ["es2015"],
-      "plugins": [
-        "transform-object-rest-spread",
-        "transform-async-to-generator",
-        "transform-runtime"
-      ],
-    },
-    "noshim": {
-      "plugins": [
-        "transform-es2015-modules-commonjs",
-        "transform-object-rest-spread",
-        "transform-async-to-generator"
-      ]
-    },
-    "firefox": {
-      "plugins": [
-        "transform-es2015-modules-commonjs",
-        "transform-object-rest-spread",
-        "transform-async-to-generator"
-      ]
```

Comment:
Ah I see.

1. `transform-es2015-modules-commonjs` is already included in the preset `es2015` (which is included in `latest`.  I added `latest` 😉 )
2. `transform-object-rest-spread` is included in `stage-3` (I included that as well)
3. same for `transform-async-to-generator` 😉 

So this is all redundant and can be removed ☺️ 

One thing: you ship this library in the browser. Ok. This is for people that reference `kinto.js` directly in their `index.html` and work with the global Kinto object. This approach is a little bit outdated, but ok, I understand that this is necessary for some people out there. For those people that **only** uses Kinto it is necessary to include the `babel-polyfill` and **all** stuff that it's necessary that it can run in the browser. For all other developers that use modern build tools like webpack and so on this workflow is not recommended.

I don't have a solution for this at the moment... Maybe we need a second build? One build for the **big** browser solution that runs out of the box and one small build for modern build tools? 🤔